### ISSUE TITLE
Clicking Update Banner Allows Multiple Update Windows To Open

### ISSUE THREAD

**Issue by m-henderson** [BobbyGR/AIOMiner#118]
AIO Miner Version: 10.30.2020

Reproduction Steps:
1). Open AIO Miner
2). Notice there is an red update banner stating you should update:
![image](https://user-images.githubusercontent.com/16070192/103288814-60f77f00-49ab-11eb-9057-62ccbf23baa1.png)
3). Clicking the update banner opens another window that will allow you to update (It should do this, this is normal):
![image](https://user-images.githubusercontent.com/16070192/103288869-7cfb2080-49ab-11eb-901f-a5d2c6b4f3b3.png)
4). Clicking the banner repeatedly opens multiple windows.  
![image](https://user-images.githubusercontent.com/16070192/103288786-5341f980-49ab-11eb-9c0e-b236d167fd0b.png)

<hr>
Expected behavior:
When clicking the update banner, the update window should be shown as well as the update banner button being disabled until that update window is closed.

**Comment by BobbyGR** [comment]
Ah, nice!  Good find. I've got a patch coming out that will resolve this and stop asking you to update.

**Comment by m-henderson** [comment]
awesome. Do you want me to leave this open until that patch is released or should we go ahead and close it?

**Comment by BobbyGR** [comment]
Leave it open to stare at me in the eyes.  :)

**Comment by m-henderson** [comment]
@BobbyGR It will be good to leave open for others who may report the issue as well. Btw, anything minor I can help with to start getting involved with this project?

**Comment by BobbyGR** [comment]
hey @m-henderson! Feel free to review the code.  I know it's hard to follow as well...it's hard to follow.  If you have time to make a settings editor that would help a ton of people.  Just load the settings file and use the AIOMinerJSON for CRUD. Should be simple-ish but just time consuming.

**Comment by m-henderson** [comment]
@BobbyGR sounds good! will see what I can do.

**Comment by BobbyGR** [comment]
https://github.com/BobbyGR/AIOMiner/pull/122
